bin/audit-svg-colors.js

```diff
@@ -0,0 +1,286 @@
+#!/usr/bin/env node
+
+/**
+ * The script looks for all colors used in the SVG images found in the repository
+ * and suggests Color Studio replacements for all non-standard values it finds.
+ *
+ * List all non-standard color values by image path:
+ * $ ./bin/audits-svg-colors.js
+ *
+ * List all non-standard color values in aggregate:
+ * $ ./bin/audits-svg-colors.js --list
+ */
```

Comment:
Is this meant to be `./bin/audit-svg-colors.js` based on the file name? I got a Permission Denied error when trying it, but whilst running this, I got a "file not found" error.